csfieldguide/curriculum_guides/content/en/apcsp/sections/global-impact.md

```diff
@@ -73,18 +86,21 @@ and aggregate levels.
 - EK 7.3.1Q Open source and free software have practical, business, and ethical impacts on widespread access to programs, libraries, and code.
 
 ### LO 7.4.1 Explain the connections between computing and real-world contexts, including economic, social, and cultural contexts.
+
 - EK 7.4.1A The innovation and impact of social media and online access varies in different countries and in different socioeconomic groups.
 - EK 7.4.1B Mobile, wireless, and networked computing have an impact on innovation throughout the world.
 - EK 7.4.1C The global distribution of computing resources raises issues of equity, access, and power.
 - EK 7.4.1D Groups and individuals are affected by the "digital divide"—differing access to computing and the Internet based on socioeconomic or geographic characteristics.
 - EK 7.4.1E Networks and infrastructure are supported by both commercial and governmental initiatives.
 
 ### LO 7.5.1 Access, manage, and attribute information using effective strategies.
+
 - EK 7.5.1A Online databases and libraries catalog and house secondary and some primary sources.
 - EK 7.5.1B Advance search tools, Boolean logic, and key words can refine the search focus, and/or limit their searches based on a variety of factors (e.g., data, peer-review status, type of publication).
```

Comment:
There's also a comma after 'e.g.' here, I guess that's how they write it. Not super important, you can make a call.